upup/pkg/fi/cloudup/BUILD.bazel

```diff
@@ -103,4 +103,7 @@ go_test(
         "//vendor/github.com/golang/glog:go_default_library",
         "//vendor/k8s.io/apimachinery/pkg/util/sets:go_default_library",
     ],
+    data = [
```

Comment:
These were autogenerated right?

Reply:
Testing stuff no. Gazelle will fix dependencies and stuff.